docs/advanced-usage/satellite-domains.mdx

```diff
@@ -256,7 +256,21 @@ You can configure your satellite application by setting the following properties
     }, options);
 
     export const config = {
-      matcher: ['/((?!.*\\..*|_next).*)', '/', '/(api|trpc)(.*)'],
+      matcher: [
+        // Skip Next.js internals and all static files, unless found in search params
+        '/((?!_next|[^?]*\\.(?:html?|css|js(?!on)|jpe?g|webp|png|gif|svg|ttf|woff2?|ico|csv|docx?|xlsx?|zip|webmanifest)).*)',
+        // Always run for API routes
+        '/(api|trpc)(.*)',
+      ],
+    };
+
+    export const config = {
+      matcher: [
+        // Skip Next.js internals and all static files, unless found in search params
+        '/((?!_next|[^?]*\\.(?:html?|css|js(?!on)|jpe?g|webp|png|gif|svg|ttf|woff2?|ico|csv|docx?|xlsx?|zip|webmanifest)).*)',
+        // Always run for API routes
+        '/(api|trpc)(.*)',
+      ],
```

Comment:
This looks like it is in here 2x, I assume this was an accident?

Reply:
Done